<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>novavendawin</class>
 <widget class="QWidget" name="novavendawin">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyOfSale - New Sale</string>
  </property>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantSpinbox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>240</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Search product:</string>
   </property>
  </widget>
  <widget class="QLabel" name="quantLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Total: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>681</width>
     <height>321</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="includeBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string>Include</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>440</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
   <property name="icon">
    <iconset theme="checkbox">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>440</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="alphaOrderCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alphabetical Order</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="customerLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="customerLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>customerLineEdit</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>quantSpinbox</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>includeBtn</tabstop>
  <tabstop>deleteBtn</tabstop>
  <tabstop>alphaOrderCheck</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>finButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>searchLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>quantSpinbox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>tableWidget</receiver>
   <slot>sortByColumn(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customerLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>includeBtn</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
